paasta_tools/cli/cmds/logs.py

```diff
@@ -1153,7 +1153,7 @@ def paasta_logs(args):
             line_offset=args.line_offset,
             levels=levels,
             components=components,
-            cluters=clusters,
+            clusters=clusters,
```

Comment:
Lol this couldn't have ever worked.

Reply:
I guess it didn't crash because log_reader takes **kwargs?